### PR TITLE
Add support of dd function

### PR DESCRIPTION
**Pull request by valentinloiseau**
Add the dd function from the list of detection functions.

### CODE REVIEW

File: extension.neon [hunk 1]
```diff
@@ -29,6 +29,7 @@ parameters:
 			-
 				type: Expr_FuncCall
 				functions:
+					- dd
```
Comment (mremi) [inline extension.neon:32]: Laravel's function right?

Comment (valentinloiseau) [reply extension.neon:32]: It's the Symfony function provided by the VarDumper Component for call dump(...) then die().
So it's probably in Laravel too.

Comment (mremi) [reply extension.neon:32]: Oh right, could you please amend your commit to update the CHANGELOG.md?

Comment (valentinloiseau) [reply extension.neon:32]: Done 👍

**Comment by mremi** [comment]
Thanks @valentinloiseau 👍

**Pull request merged**